src/main/java/me/jellysquid/mods/sodium/client/gui/utils/URLUtils.java

```diff
@@ -0,0 +1,27 @@
+package me.jellysquid.mods.sodium.client.gui.utils;
+
+import me.jellysquid.mods.sodium.client.SodiumClientMod;
+import net.minecraft.util.Util;
+
+import java.io.IOException;
+
+public class URLUtils {
+
+    private static String[] getURLOpenCommand(String url) {
+        return switch (Util.getOSType()) {
+            case LINUX, SOLARIS, WINDOWS -> new String[]{"rundll32", "url.dll,FileProtocolHandler", url};
```

Comment:
I think you meant to put only `WINDOWS` on this line.

Reply:
Oh, you are right, for Linux, Solaris and etc. we should use xdg-open